gateway/auth_manager.go

```diff
@@ -74,12 +77,32 @@ func (b *DefaultSessionManager) ResetQuota(keyName string, session *user.Session
 
 	rateLimiterSentinelKey := RateLimitKeyPrefix + keyName + ".BLOCKED"
 
-	// Clear the rate limiter
-	b.store.DeleteRawKey(rateLimiterSentinelKey)
+	// Clear the rate limiter and
 	// Fix the raw key
-	b.store.DeleteRawKey(rawKey)
+	keys := []string{rateLimiterSentinelKey, rawKey}
+	for _, acl := range session.AccessRights {
```

Comment:
shall we wrap these into a function `rawKeysWithAllowanceScope` that returns the keys to be deleted with allowance scope?

Reply:
thanks for pointing this